pages/entity/_type/_.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <b-container
+    v-if="error"
+    class="mb-3"
+  >
+    <AlertMessage
+      :error="error"
+    />
+  </b-container>
+  <b-container
+    v-else
+    data-qa="entity page"
+  >
+    <h1 data-qa="entity title">
+      {{ title }}
+    </h1>
+  </b-container>
+</template>
+
+<script>
+  import AlertMessage from '../../../components/generic/AlertMessage';
+
+  import getEntity from '../../../plugins/europeana/entity';
+
+  export default {
+    components: {
+      AlertMessage
+    },
+    data () {
+      return {
+        error: null,
+        title: null
+      };
+    },
+    asyncData ({ env, params, res, redirect }) {
+      return getEntity(params.type, params.pathMatch, {
+        wskey: env.EUROPEANA_ENTITY_API_KEY
+      })
+        .then((response) => {
+          let entityId = params.pathMatch.split('-')[0];
+          let desiredPath = entityId + (response.entity.prefLabel.en ? '-' + response.entity.prefLabel.en.toLowerCase().replace(' ', '-') : '');
```

Comment:
This only replaces the first space character which may result in URLs like `/entity/people/113592-albert-henry%20payne`. It needs to replace all space characters.